packages/server/src/stages/relative/pattern-relative-import.test.ts

```diff
@@ -17,7 +17,7 @@ describe("patternRelativeImport", () => {
   })
 
   it("matches an import statement with no specifier", () => {
-    const input = `import from "./ding"`
+    const input = `import "./ding"`
```

Comment:
`import from` isn't actual module syntax, so I changed this :D

Reply:
Nice work